default-configs/xtensa-softmmu.mak

```diff
@@ -10,6 +10,6 @@ CONFIG_XTENSA_XTFPGA=y
 CONFIG_SSP=y
 CONFIG_DW_DMA_DSP=y
 CONFIG_EDMA_DSP=y
-CONFIG_SDMA_DSP=y 
+CONFIG_SDMA_DSP=y
 CONFIG_HDA_DMA_DSP=y
```

Comment:
If you want to fix this whitespace, please create a separate patch. Because this change is not relate to bugfixing qemu boot failure :).